tests/datasets/test_xview2.py

```diff
@@ -0,0 +1,117 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import os
+import shutil
+from pathlib import Path
+from typing import Generator
+
+import pytest
+import torch
+import torch.nn as nn
+from _pytest.fixtures import SubRequest
+from _pytest.monkeypatch import MonkeyPatch
+
+from torchgeo.datasets import XView2, XView2DataModule
+
+
+class TestXView2:
+    @pytest.fixture(params=["train", "test"])
```

Comment:
```suggestion
    @pytest.fixture(scope="class", params=["train", "test"])
```

Reply:
This breaks the tests

Reply:
`ScopeMismatch: You tried to access the 'function' scoped fixture 'monkeypatch' with a 'class' scoped request object, involved factories`